docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "3"
+services:
+  bookit-api:
+    build:
+      context: .
+      dockerfile: docker/local.Dockerfile
+    command: "./gradlew bootRun"
```

Comment:
technically `bootRun` will implicitly do a `build` so there's no real need to explicitly do it in the Dockerfile... but maybe doesn't matter too much (I always get confused on the difference between command vs run

Reply:
Actually I did a build separate so Docker would cache everything before the command (it boots up in a few seconds on second deploy)